modules/tinymce/src/core/main/ts/newline/NewLineUtils.ts

```diff
@@ -104,6 +104,9 @@ const getEditableRoot = (dom, node) => {
   return parent !== root ? editableRoot : root;
 };
 
+const isBlockQuote = (editor: Editor) =>
+  SelectorFind.ancestor(SugarElement.fromDom(editor.selection.getStart(true)), 'BLOCKQUOTE', (element) => element.dom === editor.dom.getRoot()).isSome();
```

Comment:
Since this is a css selector you could use a lowercase string here. Also if we move the root predicate here to that EditorBody.ts module I mentioned that I think we should have this would be less code. Also there is a `SelectorExists.ancestor` so you don't need the isSome check here.

Reply:
I'll leave creating the editorbody.ts to a separate issue, since I feel that when we do that we should change all the other ones in the code base.